packages/cli/src/ceramic-daemon.ts

```diff
@@ -303,6 +315,11 @@ export class CeramicDaemon {
         .map(String)
         .join(', ')}`
     )
+    if (wrongIpfsClient) {
+      diagnosticsLogger.warn(
+        `The daemon expected the remote IPFS node to be running ${desiredIpfsClient} but it was running ${ipfsId.agentVersion}. The daemon config has been modified.`
```

Comment:
I think this is kind of a confusing warning for regular users.  Especially if they didn't explicitly set any "desired" ipfs client but are just using all the defaults.

Can we change this to something more along the lines of "Detected ipfs kubo running at ${url} with agent version ${agentVersion}.  Using kubo for p2p networking is deprecated - we recommend migrating to ceramic-one instead."

Reply:
adjusted to warnings